x/project/types/params.go

```diff
@@ -2,10 +2,14 @@ package types
 
 import (
 	"fmt"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/params"
-	"github.com/ixofoundation/ixo-blockchain/x/did"
-	"github.com/ixofoundation/ixo-blockchain/x/ixo"
+	params "github.com/cosmos/cosmos-sdk/x/params/types"
+	didexported "github.com/ixofoundation/ixo-blockchain/x/did/exported"
+)
+
+const (
+	IxoNativeToken = "uixo"
```

Comment:
Remove this and replace the usage of it with `ixotypes.IxoNativeToken`, with import `ixotypes "github.com/ixofoundation/ixo-blockchain/x/ixo/types"`